src/track.c

```diff
@@ -289,6 +289,17 @@ void tracking_channel_update(u8 channel)
         cs2[i].I = cs[2-i].I;
         cs2[i].Q = cs[2-i].Q;
       }
+
+      /* Output I/Q correlations using SBP if enabled for this channel */
+      if (chan->output_iq && (chan->int_ms > 1)) {
+        msg_tracking_iq_t msg = {
+          .channel = channel,
+          .sid = chan->prn,
+        };
+        memcpy(msg.corrs, chan->cs, sizeof(chan->cs));
```

Comment:
This could be a little brittle if the types of `cs` or `corrs` changes. What about assigning element by element? Another option would be to check with an `assert(sizeof(msg.corrs) == sizeof(chan->cs))`.